platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -3629,17 +3629,17 @@
                     $http.get('../mds/instances/' + relatedEntityId + '/instance/' + instanceId).success(function (data) {
                         $scope.editRelatedFields = angular.copy(data.fields);
                     }).error(function(response) {
-                        handleResponse('mds.error', 'mds.error.instancesList', response);
+                        ModalFactory.handleResponse('mds.error', 'mds.error.instancesList', response);
                     });
                 },
                 setExistingWithoutId = function () {
```

Comment:
How is this setExistingWithoutId function working within the context of the callback?